cspell/.cspell.json

```diff
@@ -46,11 +46,16 @@
         "**/*.caffemodel",
         "**/*.cu",
         "**/*.html",
+        "**/*.jpeg",
+        "**/*.jpg",
+        "**/*.mp3",
         "**/*.onnx",
         "**/*.osm",
         "**/*.pcd",
         "**/*.pt",
         "**/*.stl",
+        "**/*.svg",
```

Comment:
```suggestion
```
cSpell can detect typos in SVG; it is really helpful.
Please consider adding ignore patterns instead.
<https://github.com/tier4/autoware-spell-check-dict/blob/main/cspell/.cspell.json#L36-L42>